src/ast/passes/named_param.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "ast/pass_manager.h"
+#include "types.h"
+
+namespace bpftrace::ast {
+
+struct NamedParamDefault {
+  std::string value;
+  bool is_bool = false;
```

Comment:
Please, no. 😮‍💨 

Just do this properly, and have value be a `std::variant`. E.g.

```
std::variant<std::string, int64_t, uint64_t, bool> value;
```

Then you can also ensure that the type matches. The `type` can be carried implicitly by the variant that you've set, rather than requiring yet-another link to `SizedType` and including `types.h`.